packages/platforms/accton/x86-64/as5915-18x/Makefile

```diff
@@ -0,0 +1 @@
+include $(ONL)/make/pkg.mk
```

Comment:
Please add a newline at the end of lines. I’d be great if you configured your editor accordingly.

Reply:
Added